### ISSUE TITLE
Fenix-L10n screenshots for MKT on tablets

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/mobile-test-eng#414]
I was pinged to ask if we had screenshots on tablets for Fenix to update the Google Play Store ones with MR1 changes.

Actually we have those, see link, but only for the Tier 1 locales.
https://drive.google.com/drive/folders/1p-6WX_YwNqZ5WSm6xVBOH8zg1PgoWVQ4

This time they will need these 17 locales too:
 ar (Arabic)
 zh-CN (Simplified-China)
 zh-TW (Traditional-Taiwan)
 nl-NL (Dutch)
 hi-IN (Hindi)
 id-ID (Indonesian)
 it-IT (Italian)
 jp (Japanese)
 ko-KR (South Korea)
 pt-BR (Portuguese-Brazil)
 pt-PT (Portuguese-Portugal)
 pl-PL (Polish)
 ru-RU (Russian)
 es-419 (Spanish-Latin America)
 es-ES (Spanish-Spain)
 tr-TR (Turkey)
 vi-VI (Vietnam)

There is no rush, but so far they will use only English, nice to have all the screenshots in the Google Play localized sooner rather than later...

**Comment by isabelrios** [comment]
I have been working on this this week. We already have screenshots for Fenix, but as well as on firefox, the ones required for Marketing team are a bit different. So I'm going to create a test suite, keeping it disabled but to have the changes checked in the repo in case those are necessary in the future.

To get screenshots is easy, the problem with these is that they have to use release build so that `Firefox Browser` is shown instead of `Firefox Preview` in homescreen and the different views.
That requires a few steps:
- Generate the apk needed with this command: `./gradlew app:connectedAndroidTest -PtestBuildType=release -PdisableOptimization -Pandroid.testInstrumentationRunnerArguments.class="org.mozilla.fenix.ui.NavigationToolbarTest#visitURLTest"`
- Sign the apks, in theory the should be signed with the local.properties file key but they are not, so use: `jarsigner -keystore KEYSTORE ~Path_to_the_apk/app.apk KEYNAME`
- Start a emulator (Google API) as root: `adb root`
- Run the tests: `fastlane screengrab --test_instrumentation_runner "androidx.test.runner.AndroidJUnitRunner"`
- Get the screenshots from device: `adb pull /data/data/org.mozilla.firefox/screengrab/fr-FR/images/screenshots`

**Comment by isabelrios** [comment]
I'm still waiting for MKT feedback (Kirby) to define the tests so that they get the desired screenshots. 

Moving to Upcoming until further notice

**Comment by isabelrios** [comment]
After talking to Kirby, we have agreed to take these screenshots for those locales once MR2 is ready. 
It is fine for now to have them in EN for all tiers1 and then change to each locale once the new MR2 screenshots are required. They are going to change significantly, so even though we get them know, they will change in the near future, so let's go for the long term solution and get MR2 ones once the features are ready.
